dashboard/next.config.js

```diff
@@ -19,6 +19,7 @@
  * @type {import('next').NextConfig}
  */
 const nextConfig = {
+  output: 'export',
```

Comment:
cc @xxchan I remembered that there used to some discussion between us on this.

Reply:
This is required to upgrading to next.js v14
https://nextjs.org/docs/pages/building-your-application/upgrading/version-14#v14-summary
https://nextjs.org/docs/app/building-your-application/deploying/static-exports

Reply:
I remember `output: 'export'` will break `build-static`, and you suggested a solution `phase`

https://github.com/risingwavelabs/risingwave/pull/12421#discussion_r1330028759

Reply:
Oh, it will break debug build, because it cannot rewrite to mock api conditionally.

Reply:
Oh, you already changed it in https://github.com/risingwavelabs/risingwave/issues/14361 

I don't know whether there are any other issues now.